src/main/java/appeng/tile/networking/TileController.java

```diff
@@ -151,12 +154,10 @@ protected double funnelPowerIntoStorage( final double power, final Actionable mo
 	{
 		try
 		{
-			final double ret = this.getProxy().getEnergy().injectPower( power, mode );
-			if( mode == Actionable.SIMULATE )
-			{
-				return ret;
-			}
-			return 0;
+			final IEnergyGrid grid = this.getProxy().getEnergy();
+			final double leftOver = grid.injectPower( power, mode );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "leftOver". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1488)